src/SFA.DAS.AANHub.Api/Startup.cs

```diff
@@ -1,4 +1,7 @@
-﻿using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Cannot see what has changed, looks like things have just been moved around and reformatted?

Reply:
`app.UseFluentValidationExceptionHandler();` has been removed, but my IDE/Resharper just reformatted the file based on the standards we've discussed so far.